config/Symphony.config

```diff
@@ -1,5 +1,9 @@
 {
     "url": "https://foundation-dev.symphony.com",
     "minimizeOnClose" : false,
-    "launchOnStartup" : true
+    "launchOnStartup" : true,
+    "notifications": {
+        "position": "upper-right",
+        "display": "1234"
```

Comment:
Sorry, why do we have a 1234 here? Is this a dummy value? If so, why not keep it empty instead?